### PR TITLE
Added integration test for print and add command, updated ug 

### PR DESCRIPTION
**Pull request by Ashley-Lau**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -291,11 +293,19 @@ Format: `stats `, followed by one of the following:
...
+
 ### Saving data
 Data (all stocks in inventory in json) is automatically saved to
 `[root directory]/data/stockbook.json` when any of these commands is executed:
 * add
 * delete
+* note
+* notedelete
 * update
 * note
```
Comment (amylzting) [inline docs/UserGuide.md]: One of the note can be removed.

**Review by kaichaoang — APPROVED** [review]
LGTM, however do clear the extra "note" mentioned by @amylzting

File: src/test/java/seedu/stock/logic/commands/AddCommandIntegrationTest.java [hunk 2]
```diff
@@ -1,45 +1,46 @@
...
+        assertCommandSuccess(new AddCommand(validStock), model,
+                String.format(AddCommand.MESSAGE_SUCCESS, validStock), expectedModel);
+    }
+
+    //    @Test
+    //    public void execute_duplicateStock_throwsCommandException() {
+    //        dog test case that causes some test to fail 1/2 the times??? what kind of sorcery is this?
+    //        Stock stockInList = model.getStockBook().getStockList().get(0);
+    //        assertCommandFailure(new AddCommand(stockInList), model, AddCommand.MESSAGE_DUPLICATE_STOCK);
+    //    }
```
Comment (AudreyFelicio) [inline src/test/java/seedu/stock/logic/commands/AddCommandIntegrationTest.java]: Maybe consider removing this first so the code will be cleaner. Can always refer to this later by viewing GitHub commit history

**Review by AudreyFelicio — APPROVED** [review]
Overall LGTM! Do remove the `note` commented by @amylzting.

File: src/test/java/seedu/stock/logic/commands/AddCommandIntegrationTest.java (hunk 2 above)
Comment (Ashley-Lau) [reply src/test/java/seedu/stock/logic/commands/AddCommandIntegrationTest.java]: Alright will remove it!

**Pull request merged**